ftest-toxic-reporter/pom.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>arquillian-cube-q</artifactId>
+    <groupId>org.arquillian.cube.q</groupId>
+    <version>1.0.0.Final-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>arquillian-cube-q-reporter-ftest-toxic</artifactId>
```

Comment:
> What If I have to run all tests from all profile as a part of build? Can we run it?

I'm not sure if I understand the question. If you anyway want to have different configurations you will have to somehow swap it so this might end up with running mvn tests several times with different config. So you can either have multiple parallel builds with different profiles or just configure build to run it sequentially. But I'm not sure if I understood your concern correctly.